### DISCUSSION TITLE
Build on top of Huly?

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [hcengineering/platform#5285] [General]
We've build a CRM for HR professionals with specific focus on AI features. Huly seems great in terms both UI and UX , is it possible(and feasible) to integrate Huly with our codebase OR use it as a base and build on top of it?

**Comment by aplatoff** [comment]
Hi Tejasw,

Absolutely! Building this type of applications is one of the foundational ideas behind the Huly Platform. I'd be interested in learning more about your product for many reasons. One of them is that we also have a CRM for HR (Applicant Tracking System) built on top of the Huly Platform, so we might try to align it with your product.

Of course, you're free to build a competing product or anything different. That's perfectly OK, and we'd be happy to help you with that. Another thing you may think about is that we may consider to provide financing for development products on top of the Huly Platform (if you're interested) or think about joint venture.

Please share your thoughts.

Kind regards,
Andrey
